### ISSUE TITLE
[Bug] r.zonal.classes manual example fails

### ISSUE THREAD

**Issue by petrasovaa** [OSGeo/grass-addons#116]
**Name of the addon**
r.zonal.classes

**Describe the bug**
Example from manual fails with:
RASS 7.9.dev (nc_spm_08_grass7):/tmp > r.zonal.classes zone_map=zipcodes raster=landuse96_28m csvfile=output.csv vectormap=vect_output prefix=lu
 100%
r.stats command finished...
Traceback (most recent call last):
  File "/home/anna/.grass7/addons/scripts/r.zonal.classes", line 312, in <module>
    main()
  File "/home/anna/.grass7/addons/scripts/r.zonal.classes", line 203, in main
    mode = max(iter(totals_dict[ID].items()), key=operator.itemgetter(1))[0] 
ValueError: max() arg is an empty sequence

**Expected behavior**
This should work even if classes_list parameter is not provided. What happens now is that if you don't provide classes, you get empty dictionary that fails with the error above.

**Comment by tgrippa** [comment]
Hey @petrasovaa, 

I tried a fix in commit 9bca4e040e7ba5119405296501e641098ce4cb17 that works on my computer. Also, I improved the manual page a bit. Could you check again on your side if it runs as expected ?

**Comment by petrasovaa** [comment]
I tested the examples and yes, those work, thank you! In addition though I tested these (without the csvfile option):

`r.zonal.classes zone_map=zipcodes raster=landuse96_28m vectormap=vect_output classes_list='4,12,1234'  --o`

resulting in 

```
r.stats command finished...
Creating output vector map...
DBMI-SQLite driver error:
Error in sqlite3_prepare():
table rzonalclasses_tmp_vect_32080 has 19 columns but 18 values were supplied

DBMI-SQLite driver error:
Error in sqlite3_prepare():
table rzonalclasses_tmp_vect_32080 has 19 columns but 18 values were supplied

ERROR: Error while executing: 'INSERT INTO rzonalclasses_tmp_vect_32080
       VALUES
       (18,9e-05,0.11286,0.07422,0.00000,0.0231,0.00000,0.0549,0.00000,0.00000,0.05871,0.02337,0.00000,0.29982,0.35147,0.00146,0.00000,0.00000)'
Traceback (most recent call last):
  File "/home/anna/.grass7/addons/scripts/r.zonal.classes", line 316, in <module>
    main()
  File "/home/anna/.grass7/addons/scripts/r.zonal.classes", line 309, in main
    gscript.run_command('db.execute', input=insert_sql, quiet=True)
  File "/home/anna/dev/grass/grass/dist.x86_64-pc-linux-gnu/etc/python/grass/script/core.py", line 499, in run_command
    return handle_errors(returncode, result=None, args=args, kwargs=kwargs)
  File "/home/anna/dev/grass/grass/dist.x86_64-pc-linux-gnu/etc/python/grass/script/core.py", line 392, in handle_errors
    returncode=returncode)
grass.exceptions.CalledModuleError: Module run db.execute db.execute --q input=/home/anna/grassdata/nc_spm_08_grass7/anna/.tmp/pop-os/32080.1 ended with error

```

**Comment by petrasovaa** [comment]
One more thing, the code is works only with Python3 due to:

    with open(csvfile, 'w', newline='') as outfile:

newline is not in Python2. It's up to you if you care about that I guess.

**Comment by tgrippa** [comment]
The bug on INSERT sql statement is fixed now since 47ec9c77cf9218fd9f1d7144cf52186fc3ed265c.
For the Python 2 compatibility, I guess it is relevant to keep it as now since it is not supported anymore. 
Thanks for testing this add-on ;) If you consider the issue over, feel free to close the discussion...

**Comment by petrasovaa** [comment]
Thank you!
